### DISCUSSION TITLE
Checkout Logo Link with Headless Setup

### DISCUSSION THREAD

**Discussion by leefroese** [Shopify/storefront-api-feedback#95] [Help] [answered]
Is there a way to link the logo at checkout when using the Storefront API as a headless setup? For example: https://checkout.rugandweave.com/15988687/checkouts/dc12b45824ce0c33b27e134cc0fb3472?no_cookies_from_redirect=1 doesn't have a linked logo. It would be ideal if the logo could be linked and linked back to the headless site at  https://rugandweave.com.

**Comment by johncraigcole** [comment, marked as answer]
Hey @leefroese 👋 . This is a known issue that we hope to have a solution for in the future. Today, only a plus merchant that has access to `checkout.liquid` would be able to solve for this. I don't have any timelines on when a solution will be available but would expect when it is to be announced in the Shopify change log.

**Reply by leefroese** [reply]
Thanks! 🤞  it's soon!
